IdentityCore/src/webview/embeddedWebview/challangeHandlers/MSIDClientTLSHandler.m

```diff
@@ -30,11 +30,57 @@
 #import "MSIDCertificateChooser.h"
 #endif
 
-@implementation MSIDTLSHandler
+#if TARGET_OS_IPHONE
```

Comment:
Maybe it is better to separate MSIDClientTLSHandler.m to iOS and Mac files? We can have parent asbstract class with shared logic.

Reply:
+1

Reply:
done